vet.sh

```diff
@@ -120,11 +114,26 @@ if [[ -z "${VET_SKIP_PROTO}" ]]; then
     (git status; git --no-pager diff; exit 1)
 fi
 
-# - Check that our modules are tidy.
+# - gofmt, goimports, golint (with exceptions for generated code), go vet,
+# go mod tidy.
+# Perform these checks on each module inside gRPC.
 if go help mod >& /dev/null; then
-  find . -name 'go.mod' | xargs -IXXX bash -c 'cd $(dirname XXX); go mod tidy'
-  git status --porcelain 2>&1 | fail_on_output || \
+  for MOD_FILE in $(find . -name 'go.mod'); do
+    MOD_DIR=$(dirname $MOD_FILE)
```

Comment:
Always use braces for variables in bash scripts: `MOD_DIR=$(dirname ${MOD_FILE})`

Reply:
Done.